README.md

```diff
@@ -32,22 +31,13 @@ audited, might contain bugs and should not be used in production.
 
 ## Introduction
 
-For Aave, one of the main current and future goals is growth of liquidity and
-user base. As seen on side-chains with low transaction cost like Polygon or
-Avalanche, there is high demand to use the Aave protocol with small amounts to
-earn high yield. That's why we brought to you an initial phase of the Aave <>
-Starknet integration allowing deposit/withdrawal on Aave Ethereum by
-exclusively transacting on Starknet.
+This bridge project is the first step of Aave in Starknet ecosystem. The bridge allows users to deposit and withdraw their [aTokens](https://docs.aave.com/developers/tokens/atoken), and only `aTokens`, on StarkNet and get `static_a_tokens` instead. These `static_a_tokens` are equivalent to `aTokens`: the former grow in value when the latter grow in balance.
```

Comment:
This bridge project is the first step of Aave in Starknet ecosystem. The bridge allows users to deposit and withdraw their [aTokens](https://docs.aave.com/developers/tokens/atoken), and only `aTokens`, and receive wrapped tokens referred to as `static_a_tokens` on StarkNet. `static_a_tokens` are equivalent to `aTokens` except that the former grows in value when the latter grows in balance.

Reply:
Good suggestion, I'll adapt the sentence with your inputs